layers/cmd/valueset.go

```diff
@@ -311,6 +321,195 @@ func LoadValuesetFiles(vs *Valuesets, files []string) error {
 	return nil
 }
 
+func (opt Options) splitCell(header, cell string) []string {
+	if sep, exists := opt.Separator[header]; !exists {
+		return []string{cell}
+	} else {
+		var ret []string
+		// if separator is empty, default to space
+		sep = strings.TrimSpace(sep)
```

Comment:
Don't trim sep here, trim sep where you parse sep. Handle space there as well, so you don't have to check if sep=""  here